libraries/ui-library/src/components/six-breadcrumbs/six-breadcrumbs.scss

```diff
@@ -0,0 +1,56 @@
+@import '../../../src/global/component';
+
+:host {
+  display: block;
+  --six-breadcrumbs-separator: '/';
+  --six-breadcrumbs-background-color: var(--six-color-web-rock-200);
+  --six-breadcrumbs-padding: var(--six-spacing-x-small);
+  --six-breadcrumbs-color: pink;
+  --six-breadcrumbs-separator-color: var(--six-color-web-rock-500);
+  --six-breadcrumbs-link-color: var(--six-color-action-600);
+}
+
+.six-breadcrumbs {
+  display: block;
+  $padding: var(--six-breadcrumbs-padding);
+  background-color: var(--six-breadcrumbs-background-color);
+
+  > div {
+    /*  Container */
+
+    display: inline-flex;
+    flex-wrap: wrap;
+    width: inherit;
+    color: var(--six-breadcrumbs-color);
+    gap: $padding;
+    padding: $padding;
+
+    ::slotted(a),
+    ::slotted(div),
+    six-breadcrumbs-item {
+      &:after {
+        content: var(--six-breadcrumbs-separator);
+        color: var(--six-breadcrumbs-separator-color);
+        padding-left: var(--six-breadcrumbs-padding);
+      }
+    }
+
+    ::slotted(:last-of-type):after,
+    div:last-of-type:after,
+    six-breadcrumbs-item:last-of-type:after {
+      content: none;
+      padding-left: unset;
+    }
+
+    a,
+    ::slotted(a) {
+      color: var(--six-breadcrumbs-link-color) !important;
```

Comment:
Have you tried another way to avoid the important ? 

There is also an important set on the breadcrumbs-item component, maybe that is the origin of the issue...? 🤔💭